illustrations/pedigree.py

```diff
@@ -0,0 +1,76 @@
+import json
+from pathlib import Path
+import string
+import sys
+from types import SimpleNamespace
+
+import matplotlib.pyplot as plt
+import tskit
+
+# useful modules in the top level dir
```

Comment:
You don't need to do this - just run ``python illustrations/pedigree.py`` from the top level dir,

Reply:
right, as long as we remember that's where it's run from, I guess. On the command-line I find it easier to run the things separately from within illustrations.

Reply:
Let's move all the code for illustrations into one file, ``illustrations.py`` as suggested. Everything gets run from the root, always. Nothing to get confused about then.

Reply:
Oh yes, that's why I did it like this: it's useful to be able to copy and paste the code into a jupyter notebook, as I template and re-test most of the figure code in notebooks which is a lot more responsive than using a script.

I guess I could simply dump the code into scripts and call them within the `illlustrations.py` script. Unless you have better suggestions for easy conversion between notebook cells and free-standing scripts?

Reply:
Why can't you do that here? You can copy a function definition into a notebook, can't you?

Reply:
Oh yes, I guess I could simply turn it into a function. Good point.